### DISCUSSION TITLE
G-Helper Temp Limit And Undervolting Not Working My Laptop

### DISCUSSION THREAD

**Discussion by zeynelakintamer** [seerge/g-helper#3460] [Q&A] [answered]
My PC: Asus Rog Zephyrus G16 GA605WV

My CPU reaches 95 C Degrees easily with CPU Boost enabled/agressive in the game
Even though I set the temperature limit to 85 degrees and use undervolting, my CPU reaches 89-95 degrees.

Also, the battery health of my laptop I bought 3 weeks ago has decreased by 1%. Is it Normal?

**Comment by seerge** [comment, marked as answer]
@zeynelakintamer temp limit is a throttle temp (default value is 96C), means CPU will drop power when it reaches this temp. If you have a very bad chip from the factory or bad liquid metal and it overheats - it will keep overheating still. It doesn't mean undervolting it's not working.

Don't forget that undervolting is not supposed to drop temps at all :) It only improves performance for same power draw.

You can try to set custom lower power limits (like 30W) to tackle temps

**Comment by zeynelakintamer** [comment]
hmm. but isn't this photo for setting the temperature limit?
If not, what techniques should I use to drop temps?
![image](https://github.com/user-attachments/assets/070d29ca-6bc4-47da-a300-35e9d26cc406)

**Reply by zeynelakintamer** [reply]
@seerge sorry I don't know using github. I missed use mention

**Reply by seerge** [reply]
@zeynelakintamer if CPU is already overheated, it may till exceed temp limit (or if GPU generates excessive heat as well). Don't forget to check `Auto Apply` underneath btw, as AMD driver may override that sometimes.

Other way to drop temp - is just set a CPU power limit in advance, like to 30W as I mentioned above. For typical gaming scenarios it's usually more than enough anyway

**Reply by zeynelakintamer** [reply]
I understand. thank u.
